packages/function-tree/src/index.js

```diff
@@ -7,6 +7,8 @@ import PropsProvider from './providers/Props'
 import PathProvider from './providers/Path'
 import Path from './Path'
 import {Sequence, Parallel, Primitive} from './primitives'
+import {FunctionTreeExecutionError, FunctionTreeError} from './errors'
+export {FunctionTreeExecutionError, FunctionTreeError} from './errors'
```

Comment:
Probably we should leave only reexports in index and move actual implementation to separate file